lib/container.ts

```diff
@@ -402,6 +408,10 @@ export class Container extends (EventEmitter as {
 				});
 		});
 	}
+
+	private getBuildArgsForDockerApi(): string[] {
+		return _.map(this.buildArguments, (v, k) => `${k}=${v}`);
```

Comment:
Does it work (is it supposed to...) if the key or value contains space characters or other "special" characters? (There was a support thread the other day... it was a supervisor validation issue I think, but the customer was using env vars with certain characters that were not being allowed.) 

Maybe update the test case to include some such characters.

Reply:
Good point, added a test!